x-pack/legacy/plugins/epm/server/lib/agent/tests/input.yml

```diff
@@ -0,0 +1,7 @@
+type: log
+paths:
+{{#paths}}
+  - "{{&.}}"
```

Comment:
The `&` is need to make sure the path is not encoded. Need to check if I could do this differently so it is not exposed to the user. We should never encode.